Dockerfile

```diff
@@ -57,6 +57,11 @@ RUN if [ "$MODEL_TYPE" = "sdxl" ]; then \
       wget -O models/vae/sdxl-vae-fp16-fix.safetensors https://huggingface.co/madebyollin/sdxl-vae-fp16-fix/resolve/main/sdxl_vae.safetensors; \
     elif [ "$MODEL_TYPE" = "sd3" ]; then \
       wget --header="Authorization: Bearer ${HUGGINGFACE_ACCESS_TOKEN}" -O models/checkpoints/sd3_medium_incl_clips_t5xxlfp8.safetensors https://huggingface.co/stabilityai/stable-diffusion-3-medium/resolve/main/sd3_medium_incl_clips_t5xxlfp8.safetensors; \
+    elif [ "$MODEL_TYPE" = "flux1-schnell" ]; then \
+      wget -O models/unet/flux1-schnell.safetensors https://huggingface.co/black-forest-labs/FLUX.1-schnell/resolve/main/flux1-schnell.safetensors && \
```

Comment:
You'll probably want to add `  unet: models/unet/` to the src/extra_model_paths.yaml

Reply:
@hoblin that makes sense, will add this. 

My overall plan is to also have an image with flux1-dev, I just ran out of time testing the basic version with just FLUX.1-schnell. But adding `unet` into the mapping is still valuable anyway.

Reply:
BTW if you have problems running `-basic` container you might try this fix https://github.com/blib-la/runpod-worker-comfy/issues/54#issuecomment-2282829872